wsapi/wsapiStructs.go

```diff
@@ -273,6 +273,7 @@ type AddressRequest struct {
 
 type HeightRequest struct {
```

Comment:
Consider adding a function:

func (h HeightRequest) IncludeRaw() bool {
    return !h.NoRaw 
}

The problem is that booleans that are the negation of what you need can be very confusing.